numpy/lib/tests/test_function_base.py

```diff
@@ -3683,30 +3683,32 @@ def test_quantile_scalar_nan(self):
     def test_quantile_identification_equation(self, method, alpha):
         # Test that the identification equation holds for the empirical
         # CDF:
-        #   E[V(x, Y)] = 0  <=>  x is quantile
+        #   dE[V(x, Y) * (x - y)]/dx = 0  <=>  x is quantile
```

Comment:
This is not correct. The old formula is correct, i.e. identification functions can be used as generalized L-Estimators.

Reply:
I went by Eq. 6 of the Gneiting paper, and I presume you went by Eq. 13?

Reply:
Yes. 6 is for scoring functions, 13 for identification functions.